ubtue_Davarlogos.js

```diff
@@ -70,33 +71,23 @@ function postProcess(doc, item) {
 				item.tags.push(tags[i].replace(/^\w/gi,function(m){return m.toUpperCase();}));
 		}
 	}
-	
-	if (!item.abstractNote) {
-		item.abstractNote = ZU.xpath(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "item abstract", " " ))]');
-		if (item.abstractNote && item.abstractNote.length > 0)
-			item.abstractNote = ZU.unescapeHTML(item.abstractNote[0].textContent.trim().replace('Resumen', ''));
-	}
-	
-	
-	//scrape item.pages
-	//ToDo item.pages = attr(doc, '.pages')
-	
+	item.title = item.title.replace(' | DavarLogos', '');
 	item.volume = ZU.xpathText(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "title", " " ))]').trim().substr(4, 7).replace('Núm', '');
 	item.issue = ZU.xpathText(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "title", " " ))]').trim().substr(-20, 1);
 	item.date = ZU.xpathText(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "title", " " ))]').trim().substr(-17, 4);
 	item.url = attr(doc, '.pdf', 'href');
 	item.ISSN = '1666-7832'
-	item.itemType = 'journalArticle'
-	
-	item.complete();
+	//ToDO scrape item.pages before getSearchResults see e.g. fzpth
```

Comment:
Open TODO

Reply:
so solution found. you wanted to look for a workaround.